conversion/conversion.go

```diff
@@ -103,7 +103,7 @@ func SchemaConv(driver string, targetDb string, ioHelper *IOStreams, schemaSampl
 	}
 }
 
-func DataConv(driver string, ioHelper *IOStreams, client *sp.Client, conv *internal.Conv, dataOnly bool) (*spanner.BatchWriter, error) {
+func DataConv(driver, sqlConnectionStr string, ioHelper *IOStreams, client *sp.Client, conv *internal.Conv, dataOnly bool) (*spanner.BatchWriter, error) {
```

Comment:
This is okay for now, but should be cleaned up to be replaced with *cmd.SourceProfile such that the same can be nil when running in legacy mode. But the issue here is that cmd package depends on conversion package, and this would cause a circular dependency. Thus, happy to defer this cleanup for later.

Reply:
Yes will file an issue to track.